src/babylon.assetContainer.ts

```diff
@@ -115,5 +199,50 @@ module BABYLON {
                 this.scene.mainSoundTrack.RemoveSound(o);
             })
         }
+        
+        private _moveAssets<T>(sourceAssets: T[], targetAssets: T[], keepAssets: T[]): void {
+            for (let asset of sourceAssets) {
+                let move = true;
+                for (let keepAsset of keepAssets) {
+                    if (asset === keepAsset) {
+                        move = false;
+                        break;
+                    }
+                }
+    
+                if (move) {
+                    targetAssets.push(asset);
+                }
+            }
+        }
+
+        /**
+         * Removes all the assets contained in the scene and adds them to the container.
+         * @param keepAssets Set of assets keep in the scene. (default: empty)

```

Comment:
kept ? (I'm sure I'm wrong :))

Reply:
Will fix: Set of assets to keep in the scene. (default: empty)